neighbors/src/components/AddPost/AddPostLayout.css

```diff
@@ -72,17 +74,60 @@
 
 textarea {
   min-height: 150px;
+  resize: none;
 }
 
 .btn {
   cursor: pointer;
 }
 
+@media screen and (max-width: 1400px) {
+  .bg-gradient {
+    background-size: 200%;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .contact-box {
+    grid-template-columns: 1fr;
+  }
+  .left {
+    width: 100%;
+    margin: 0;
+  }
+  .post-box {
+    padding-top: 50px;
+    grid-template-columns: 1fr;
+  }
+}
+
 @media screen and (max-width: 880px) {
   .contact-box {
     grid-template-columns: 1fr;
   }
   .left {
-    height: 200px;
+    width: 100%;
+    height: 100%;
+    margin: 0;
```

Comment:
tego juz nie trzeba bo poprzednie media query `@media screen and (max-width: 900px) { `
juz to zapewnia :) tak samo jak width 100% mozna usunac bo juz tez sie zawiera w poprzednim